src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -51,7 +51,8 @@ class ProjectGeneralSettings extends AsyncView {
     const {orgId, projectId} = this.props.params;
     return [
       ['data', `/projects/${orgId}/${projectId}/`],
-      ['groupingConfigs', '/grouping-configs/'],
+      ['groupingConfigs', `/grouping-configs/`],
```

Comment:
Do these requests get made if you don't have feature flag?

Reply:
yeah. should probably optimize this

Reply:
Resolved.